lib/plottr_components/src/components/project/BookList.js

```diff
@@ -145,6 +147,14 @@ const BookListConnector = (connector) => {
       })
     }
 
+    renderBookDialog = () => {
+      const { bookDialogBookId, bookDialogBookNumber, isBookDialogVisible } = this.props
+
+      if (!isBookDialogVisible) return false
+
+      return <BookDialog bookId={bookDialogBookId} bookNumber={bookDialogBookNumber} />
```

Comment:
Please select the `bookDialogBookId` and `bookNumber` from the `BookDialog` rather than passing them down.